components/contribution-flow/StepCheckout.js

```diff
@@ -47,25 +46,17 @@ const StepCheckout = ({ stepDetails, order }) => {
               includeMargin
             />
             <P mb="16px">{order.paymentMethod.data.depositAddress}</P>
+            <StyledButton onClick={() => copy(window.location.href)} disabled={isCopied}>
```

Comment:
Copying `window.location.href`? What is this?

Reply:
"Click to copy wallet address"

Reply:
My mistake. This should be copying the wallet address. I've corrected that. 👍🏽